Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -75,6 +76,9 @@ protected override void Start()
                     responses.Add(keyword, keywordAndResponse.Response);
                 }
             }
+
+            rememberFocusRequired = this.IsFocusRequired;
```

Comment:
Could you explain why you are needing to save off IsFocusRequired? Is it beause of how the eye tracking example scenes are registering for speech commands? My understanding is "hide cursor" would be a global speech command handled by a global handler so I'm confused why we need to do this save / restore here.